zoom_monitor.py

```diff
@@ -0,0 +1,74 @@
+from contextlib import contextmanager
+import time
+
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver import Chrome
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@contextmanager
+def monitor_zoom(url):
+    zoom = ZoomMonitor(url)
+    try:
+        yield zoom
+    finally:
+        zoom.clean_up()
+
+
+class ZoomMonitor():
+    def __init__(self, url):
+        self.driver = Chrome()
+
+        # The links that we receive prompt you to open the Zoom app if it's
+        # available. Replace the domain name to skip that.
+        updated_url = url.replace("viam.zoom.us/j", "app.zoom.us/wc/join")
```

Comment:
This is open-source, and maybe people outside of Viam will want to use it. Change this to be more general.